sdk/identity/identity/src/credentials/chainedTokenCredential.ts

```diff
@@ -60,7 +60,7 @@ export class ChainedTokenCredential implements TokenCredential {
     let successfulCredentialName = "";
     const errors = [];
 
-    const { span, updatedOptions } = createSpan("ChainedTokenCredential-getToken", options);
+    const { span, updatedOptions } = createSpan("ChainedTokenCredential.getToken", options);
```

Comment:
Was there an issue for this or did you happen to notice it was wrong?

Reply:
I’d say, let’s not make this change here unless we make it everywhere in Identity. An issue could help.

Reply:
On second thought, @KarishmaGhiya can we fix this everywhere else that you find it? Or is this the only place?

Reply:
so I found inconsitencies in use of "." and "-" with getToken @sadasant

Reply:
https://github.com/Azure/azure-sdk-for-js/issues/16806 Filed an issue. May be I can address this later? @chradek @sadasant

Reply:
@chradek I happened to notice the inconsistencies

Reply:
Yes, let’s thing about this later. Also, we should align with what other clients are doing! Or align other packages to something that makes sense, (either `-` or `.`).